Parser/variables.json

```diff
@@ -0,0 +1,7066 @@
+{
```

Comment:
Is this the file stated in mylang.js ( varFilePath = 'variable_and_tables_info/variables.json') ?